ocs_ci/ocs/ui/views.py

```
@@ -1867,16 +1867,22 @@
+    "single-perspective": (
+        "//*[@id='only-one-perspective']",
+        By.XPATH,
+    ),
     "all-clusters_dropdown": (
         "//button[@data-test-id='perspective-switcher-toggle']",
         By.XPATH,
     ),
     "local-cluster_dropdown_item": (
-        "//button[@data-test-id='perspective-switcher-toggle']//*[normalize-space()='local-cluster']",
+        "//*[@data-test-id='perspective-switcher-toggle']//*[normalize-space()='local-cluster'] | "
+        "//*[@data-test-id='perspective-switcher-toggle']//*[normalize-space()='Core platform']",
```

Ruff (0.16.1)

[warning] Unparenthesized implicit string concatenation in collection

Did you forget a comma?

(ISC004)